docs/doc-config.js

```diff
@@ -32,14 +32,13 @@ module.exports = {
         { from: 'datagrid/header', to: 'grid/header' },
         { from: 'datagrid/events', to: 'grid/events' },
         { from: 'datagrid/performance', to: 'grid/performance' },
-        { from: 'datagrid/style-by-css', to: 'grid/style-by-css' },
         { from: 'datagrid/accessibility', to: 'grid/accessibility' },
         { from: 'datagrid/datagrid-v2-migration', to: 'dashboards/grid-migration' },
         { from: 'datagrid/datagrid-with-angular', to: 'grid/wrappers/grid-with-angular' },
         { from: 'datagrid/datagrid-with-react', to: 'grid/wrappers/grid-with-react' },
         { from: 'datagrid/datagrid-with-vue', to: 'grid/wrappers/grid-with-vue' },
-        { from: 'dashboards/dashboards-with-angular', to: 'dashboards/wrappers/grid-with-angular' },
-        { from: 'dashboards/dashboards-with-react', to: 'dashboards/wrappers/grid-with-react' },
-        { from: 'dashboards/dashboards-with-vue', to: 'dashboards/wrappers/grid-with-vue' }
+        { from: 'dashboards/dashboards-with-angular', to: 'grid/wrappers/grid-with-angular' },
+        { from: 'dashboards/dashboards-with-react', to: 'grid/wrappers/grid-with-react' },
+        { from: 'dashboards/dashboards-with-vue', to: 'grid/wrappers/grid-with-vue' }
```

Comment:
Dashboards should be like they were.

Reply:
They don't exist though https://www.highcharts.com/docs/wrappers/grid-with-angular, which makes the doc-builder error

Reply:
Hmm, is this a bug? The .md files and their declaration in sidebars.js exist.

<img width="292" alt="image" src="https://github.com/user-attachments/assets/3c982f6f-c0de-4229-ae78-3ef8ef42c975" />

<img width="535" alt="image" src="https://github.com/user-attachments/assets/eb28c01d-72fe-4b9a-9d15-ac15c3b1a50a" />

Reply:
`dashboards/dashboards-with-angular` should be redirected to https://www.highcharts.com/docs/dashboards/wrappers/dashboards-with-angular (it exists)

The same as react/vue.